travis/shared_configs/docs-build.yml

```diff
@@ -14,24 +14,28 @@ jobs:
         use: conda
       install: skip
       before_script:
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
-        - conda config --set always_yes yes --set changeps1 no
-        - conda config --set channel_priority strict
-        - conda config --remove channels defaults
-        - conda config --add channels pcds-tag
-        - conda config --add channels conda-forge
-        - conda config --add channels "file://`pwd`/bld-dir"
+        - curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+        - eval "$(./bin/micromamba shell hook --shell=bash)"
+        - mkdir ~/micromamba
```

Comment:
What's this for?

Reply:
micromamba needs a target directory akin to ~/miniconda3 to store the environments it creates. The default path is ~/micromamba and it does not create the directory itself. There must be a built-in command to initialize this but I didn't figure out all the details.

Reply:
I see - that makes sense. It may be worth adding a comment to that effect.

Reply:
Easy to add a few comments :+1: